test/navigation/NavigatorSpec.scala

```diff
@@ -301,258 +299,6 @@ class NavigatorSpec extends SpecBase {
 
     }
 
-    "in Check mode" - {
```

Comment:
How come we removing all this? - Shouldn't this still be working?

Reply:
I moved all the checkmode naigation test into **test/navigation/CheckModeNavigatorSpec.scala**
the file was getting particularly long and thought it would be best split between the modes